force-app/main/default/lwc/pd1CertificationCalculator/pd1CertificationCalculator.js

```diff
@@ -0,0 +1,74 @@
+import { LightningElement } from 'lwc';
+const passingScore =  68;
+export default class Pd1CertificationCalculator extends LightningElement {
+   
+    devFundamentalsScore = 23;
+    testDebugDeployScore = 22;
+    processAutomationScore  = 30;
+    userInterfaceScore  = 25;
+   
+    certificationscore = 0;
+    numberOfExamQuestions = 60;
+
+    attemptHistory = [];
+    
+    showResources = false;
+    showGoodJob = false;
+
+     currentHistoryId = 0;
+    
+     calculatescore(){
+        let devfundamentalweight = this.devFundamentalsScore * 0.23;
+        let processautomationweight = this.processAutomationScore * 0.3;
+        let userinterfaceweight = this.userInterfaceScore * 0.25;
+        let testdebugdeployweight = this.testDebugDeployScore * 0.22;
+        this.certificationscore = Math.floor(devfundamentalweight  + processautomationweight  + userinterfaceweight + testdebugdeployweight);
+
+        this.showResourcesTest();
+        this.addAttemptHistory(this.certificationscore);
+     }
+
+     handleChange(event){
+        const inputName = event.target.name;
+        let value = Number(event.target.value);
+        if(inputName === 'devFundamentals'){
+            this.devFundamentalsScore  = value; 
+        } else if(inputName === 'processAutomation'){
+            this.processAutomationScore = value; 
+        } else if(inputName === 'userInterface'){
+            this.userInterfaceScore = value; 
+        } else if(inputName === 'testDebugDeploy'){
+            this.testDebugDeployScore = value; 
+        }
+    }
+
+        showResourcesTest(){
+            if(this.certificationscore < passingScore){
+                this.showResources = true;
+                
+            } else{
+                this.showResources = false;   
+            }
+            this.showGoodJob = !this.showResources;
```

Comment:
```suggestion
            if (this.certificationscore < passingScore) {
                this.showResources = true;
            } else {
                this.showResources = false;   
            }
            this.showGoodJob = !this.showResources;
```
Watch your spacing and formatting.

Try using a getter method for this.

https://developer.salesforce.com/docs/platform/lwc/guide/js-props-getters-setters.html